beets/library.py

```diff
@@ -607,6 +607,20 @@ def from_path(cls, path):
         i.mtime = i.current_mtime()  # Initial mtime.
         return i
 
+    @classmethod
+    def construct_match_queries(cls, **info):
+        subqueries = []
+        for (key, value) in info.items():
```

Comment:
I don't think the parentheses are necessary.
```suggestion
        for key, value in info.items():
```